assets/javascripts/initializers/discourse-calendar.js.es6

```diff
@@ -298,6 +307,35 @@ function initializeDiscourseCalendar(api) {
       }
     });
   }
+
+  function _getTimeZone($calendar, currentUser) {
+    let defaultTimezone = $calendar.attr("data-calendar-default-timezone");
+    const isValidDefaultTimezone = !!moment.tz.zone(defaultTimezone);
+    if (!isValidDefaultTimezone) {
+      defaultTimezone = null;
+    }
+
+    const currentUserTimezone = currentUser ? currentUser.timezone : null;
+    return defaultTimezone || currentUserTimezone || moment.tz.guess();
```

Comment:
Maybe this instead of the two lines, especially given first line will always get called even if we have a defaultTimezonz?

```suggestion
    return defaultTimezone || (currentUser && currentUser.timezone) || moment.tz.guess();
```